packages/hash/api/src/graphql/typeDefs/knowledge/page.typedef.ts

```diff
@@ -22,6 +22,14 @@ export const knowledgePageTypedef = gql`
     The contents of the page.
     """
     contents: [KnowledgeBlock!]!
+    """
+    The index of the page in the page tree.
```

Comment:
Is this the fractional index? If so should we be explicit about that

Reply:
It is, yes :)

Reply:
Addressed in https://github.com/hashintel/hash/pull/1157/commits/5c71f70bc5f067f1883fabb47fa7ccddc373e8f3